packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -217,14 +220,14 @@ const _generateStyles = (inline: boolean, light: boolean) => {
 
     const focusStyling = {
         ...focusStyles.focus[":focus-visible"],
-        borderRadius: border.radius.radius_040,
+        borderRadius: border.radius.radius_010,
         outlineOffset: border.width.medium,
     };
 
     const pressStyling = {
         color: variant.press.foreground,
         textDecoration: "underline currentcolor solid",
-        textUnderlineOffset: theme.root.border.offset,
+        textUnderlineOffset: theme.root.font.textDecoration.offset,
```

Comment:
The text underline properties can use the new text decoration global tokens directly since it's the same in both themes!